langroid/agent/chat_agent.py

```diff
@@ -697,6 +1044,10 @@ def llm_response(
             if isinstance(message, str)
             else message.metadata.tool_ids if message is not None else []
         )
+
+        # If using strict output format, parse the output JSON
+        self._load_output_format(response)
```

Comment:
since `llm_response_messages` is already doing this, do we need this here? Or is there an execution path where this is needed?

Reply:
Removed (similarly in the async case)